src/user-guide/installing-workflows.rst

```diff
@@ -64,6 +64,12 @@ Any of the above commands may be run from anywhere on the file system with the
 addition of the option ``--directory=PATH/TO/SOURCE/DIRECTORY`` (alternatively,
 ``-C PATH/TO/SOURCE/DIRECTORY``).
 
+.. note::
+
+   To avoid confusion install will fail if ``--flow-name=runN`` or
+   ``--flow-name=run<number>``. These values are reserved for naming runs.
```

Comment:
Suggestion to make it clear it's disallowed anywhere in the workflow name (not just when using `--flow-name`)
```suggestion
   To avoid confusion, ``cylc install`` will fail if the workflow name
   contains ``runN`` or ``run<number>``. These directory names are
   reserved for the automatic numbering of run directories.
```